client/pages/sections/protocols/steps.js

```diff
@@ -37,16 +38,25 @@ class Step extends Component {
     this.props.moveDown();
   }
 
+  componentDidMount() {
+    if (this.props.parts && this.props.parts.length === 5) {
+      const activeStep = parseInt(this.props.parts[3], 10);
```

Comment:
There's a lot of magic numbers here. What's the significance of 5 & 3?

Reply:
I was hoping you wouldn't notice that...

Reply:
👀